README.md

```diff
@@ -47,6 +47,11 @@ danger(args) {
 
 ### Setup
 
+### Docker
+```sh
+docker run --rm -v "${PWD}:/code" -w /code ghcr.io/danger/danger-kotlin:1.3.3 danger-kotlin local
```

Comment:
This would theoretically also work without changing the work directory, but the default dir is the root, so mounting the current working directory there could override existing files. Mounting it in a new dir and using that for running the command seems much safer.